src/components/lists/TokenListItem.vue

```diff
@@ -11,11 +11,25 @@
     />
     <div class="flex-auto">
       {{ token.symbol }}
-      <div class="text-gray text-sm w-40 md:w-60 truncate">
+      <BalTooltip
+        v-if="isWalletReady"
```

Comment:
That is correct. I decided not to check for MM specific because more & more wallets have added or have been requested to add the 'add to token' functionality.